Jinhee/models/postDao.js

```diff
@@ -0,0 +1,74 @@
+const appDataSource = require('./dataSource.js')
+
+const createPost = async (userId, title, content) => {
+  try {
+    return await appDataSource.query(
+      `INSERT INTO posts(
+		    user_id,
+		    title,
+		    content
+		) VALUES (?, ?, ?);
+		`,
+      [userId, title, content]
+    )
+  } catch (err) {
+    const error = new Error('INVALID_DATA_INPUT')
+    error.statusCode = 500
+    throw error
+  }
+}
+
+const getPost = async () => {
+  try {
+    return await appDataSource.query(
+      `SELECT
+      users.profile_image as userProfileImage,
+      posts.user_id as userId,
+      posts.id as postingId,
+      posts.image_url as postingImageUrl,
+      posts.content as postingContent
+    FROM posts
+      INNER JOIN users ON posts.user_id = users.id`
+    )
+  } catch (err) {
+    const error = new Error('INVALID_DATA_POSTS')
+    error.statusCode = 500
```

Comment:
상태코드 500은 백엔드가 에러처리를 안 했다는 것이 되기 때문에 400번대 상태코드로 바꿔주세요.
(전체적으로 다 확인해 주세요.)
```suggestion
    error.statusCode = 400
```